distribution/ltp-upstream/include/runtest.sh

```diff
@@ -358,3 +373,11 @@ GetFailureLog ()
     local parser=$thisdir/ltp_log_parser.py
     python3 $parser -F -t 0 $logfile
 }
+
+GetAllLog ()
+{
+    local logfile=${1?"*** log file ***"}
```

Comment:
I don't understand this, what does it do? At first I thought it was `${1:?...}`, but it's not. This seems to... complain if `$1` is not set? Or something?

Anyway, once this is clarified or resolved, should I merge?

Reply:
Hi @emanchado, yes, `${1?"*** log file ***}` is to make sure a log file is passed as $1.
I'm asking @rasibley to take a look at the commit, if she also thinks it's okay, please help to merge it, thanks!
BTW, for the test results, please review the Jira ticket `FASTMOVING-1466`.